gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+|------|--------|------
+| is_hearing_impaired | bool | Indicates that the Rider has a partial or total hearing impairment
+| is_vision_impaired | bool | Indicates that the Rider has a partial or total vision impairment
+| wheelchair_accessibility | enum* | If specified, one of `ramp`, indicating that the Rider requires a wheelchair ramp in order to enter or exit the Vehicle, or `collapsible` to indicate that no ramp is required, but that the associated Vehicle needs enough storage space to carry a collapsed wheelchair. Other variants may be added to this enum.
+
+### Vehicle Fields
+
+| Name | Type | Description
+|------|--------|------
+| id | string | The unique identifier for this vehicle
+| drivers | string[] | The Person(s) who are responsible for communicating with Riders of this vehicle and for its operation. In most cases this will be a singleton array, but it is possible that groups of people will together fulfill the driver role (example: One drives while the other navigates)
+| seating | VehicleSeat[] | An array of Seating values indicating how the passenger seating is configured
+| availability | Interval[] | An array of times during which this vehicle is operational and can be assigned to Demands.
+| range | Duration | The maximum amount of time this vehicle can travel for any particular ride. This value might be the minimum of the furthest the vehicle's driver is willing to travel, or of the maximum range of the vehicle itself, such as with an electric vehicle subject to frequent recharging stops.
+| location | Location | This vehicle's location, from which it is assumed to begin or end with each availiability interval.
+
+#### VehicleSeat
+
+| Name | Type | Description
+|------|--------|------
+| type | enum | A value indicating the type of seat. Known values include `standard`, indicating that the seat is suitable for an able-bodied adult; `middle` indicating that the seat is a middle seat, and `wheelchair`, indicating that the seat is intended to be occupied directly by a lockable, electric wheelchair and nothing else.
+
+Note that this type only has a single field at the moment, but we choose to model it as a structured type with one field rather than embedding the enumerated field directly where this type is used, to allow for the possibility of adding new, optional fields in the future.
+
+### Assignment Fields
+
+An Assignment indicates a specific matching between a Vehicle and a Demand.
+
+An implementation supporting the APIs here would employ some matching and scheduling algorithm to pair Demands with Drivers and Vehicles in order to make specific Assignments, in which a vehicle is expected to satisfy a Rider's Demand by picking up and dropping off at a specified time and place. Assignments may be removed, such as by the cancellation by a Rider or a Driver. Doing so does not affect the underlying Demand, which may still be satisfied by the scheduling algorithm by making an alternate Assignment.
```

Comment:
For “Assignments may be removed, such as by the cancellation by a Rider or a Driver” 
add also “Dispatcher or Administrator”